codechecker/CodeChecker.py

```diff
@@ -340,6 +340,13 @@ def signal_handler(sig, frame):
                                         ' and defines and pass them to Clang.'
                                         'This is useful when you'
                                         'do cross-compilation.')
+        qcheck_parser.add_argument('--export-plist',
```

Comment:
Could we use other command line argument here like `-o, --output` and some separate output `format/type` argument (`--output-type` maybe).
With that we get more flexibility in the future if we want to provide not only `plist` output.